src/views/ProfileView.vue

```diff
@@ -0,0 +1,79 @@
+<script>
+import { PROFILE_URL } from "../utility/UrlConstants.js";
+import {
+  appendAuthenticationHeaders,
+  getUsername,
+} from "../utility/AuthenticationUtility.js";
+
+export default {
+  data() {
+    return {
+      userProfileData: null,
+      errorMsg: null,
+      isDataLoading: false,
+    }
+  },
+  methods: {
+    displayUsername() {
+      return getUsername();
+    },
```

Comment:
this is not required. In the api response, we are getting the username, so you can directly use that.